tests/code_wars/test_scramble.py

```diff
@@ -1,4 +1,4 @@
-from code_challenges.code_wars.scramble import scramble, unscramble
+from code_challenges.code_wars.scramble import count_characters, scramble
```

Comment:
Which of these imports is used.  Only the ones used should be imported.

Also, write unit tests for each function you write, not just the outside function.

So in this case you do need the import since the new tests you write will need it.